content/docs/command-reference/params/index.md

```diff
@@ -24,8 +24,9 @@ dependencies: _parameters_. Parameters are defined using the the `-p`
 
 In contrast to a regular <abbr>dependency</abbr>, a parameter is not a file (or
 directory). Instead, it consists of a _parameter name_ (or key) to find inside a
-YAML, JSON, or TOML _parameters file_. Multiple parameter dependencies can be
-specified from one or more parameters files.
+YAML (specifically in YAML 1.2 format), JSON, or TOML _parameters file_.
+Multiple parameter dependencies can be specified from one or more parameters
+files.
```

Comment:
Same, just "YAML 1.2"? (Linked)

Reply:
Also, there's several other files that include this params file formats list. Please aee https://github.com/iterative/dvc.org/pull/1799#pullrequestreview-494005257

Reply:
In fact I'd just grep `YAML` (upper case and double check all instances (there's 28).

Reply:
Let's not change everywhere, it's enough to mention it in key areas. It's still YAML, so it's fine.

Reply:
Same here, just Same, just "YAML 1.2" please.

> it's enough to mention it in key areas

Why is the params index more important than docs/start/experiments or docs/user-guide/basic-concepts/parameter?

Reply:
> In fact I'd just grep YAML (upper case and double check all instances

I didn't mean change it everywhere, but we need to check all instances and decide which ones to change. I think it's definitely more than just here.